diff -Nur eel-1.0.1.orig/eel/eel-font-manager.c eel-1.0.1/eel/eel-font-manager.c
--- eel-1.0.1.orig/eel/eel-font-manager.c	Tue Jul  3 18:51:17 2001
+++ eel-1.0.1/eel/eel-font-manager.c	Fri Aug  3 16:56:19 2001
@@ -55,7 +55,7 @@
 #define FONTS_SCALE_FILE_NAME		"fonts.scale"
 
 #define POSSIBLE_FONT_SERVER_CONFIG_FILES "/etc/X11/fs/config,/etc/xfs.conf"
-#define DEFAULT_FONT_DIRECTORY		EEL_DATADIR "/fonts/urw"
+#define DEFAULT_FONT_DIRECTORY		"/usr/share/fonts/Type1"
 #define USER_FONT_DIRECTORY_NAME	"fonts"
 
 #define EEL_FONT_UNDEFINED		((EelFontType) 0)
